packages/engines/snakemake/Dockerfile

```diff
@@ -1,6 +1,8 @@
 ARG SNAKEMAKE_VERSION
-FROM public.ecr.aws/amazonlinux/amazonlinux:2 AS final
+FROM 680431765560.dkr.ecr.us-west-2.amazonaws.com/aws/agc-snakemake:latest AS build
 
+FROM public.ecr.aws/amazonlinux/amazonlinux:2 AS final
+COPY --from=build  opt/work/snakemake/ /opt/work/snakemake/
```

Comment:
Just double checking, is the build directory supposed to be relative path?

Reply:
good find. It worked alright but fixed to best practice. Thanks!